jackson/src/main/java/com/lightbend/lagom/javadsl/jackson/JacksonSerializerFactory.java

```diff
@@ -33,18 +33,18 @@
 @Singleton
 public class JacksonSerializerFactory implements SerializerFactory {
 
-    private final JacksonObjectMapperProvider objectMapperProvider;
+    private final ObjectMapper objectMapper;
 
     @Inject
     public JacksonSerializerFactory(ActorSystem system) {
-      objectMapperProvider = JacksonObjectMapperProvider.get(system);
+        this(JacksonObjectMapperProvider.get(system).objectMapper());
     }
 
     /**
      * For testing purposes
      */
-    public JacksonSerializerFactory(JacksonObjectMapperProvider provider) {
-      objectMapperProvider = provider;
+    public JacksonSerializerFactory(ObjectMapper objectMapper) {
```

Comment:
Technically, this changes an API, but since it was explicitly marked as "for testing purposes" I thought that should be OK. Is it?